modular_splurt/code/datums/traits/negative.dm

```diff
@@ -125,42 +125,84 @@
 
 /datum/quirk/dumb4cum
 	name = "Dumb For Cum"
-	desc = "For one reason or another, you're totally obsessed with cum. The heat of it, the smell... its taste... It's quite simply euphoric."
+	desc = "For one reason or another, you're totally obsessed with seminal fluids. The heat of it, the smell... the taste... It's quite simply euphoric."
 	value = 0
-	gain_text = "<span class='notice'>You suddenly start craving some seed inside of you!<span>"
-	lose_text = span_danger("Cum didn't even taste that good, anyways.")
+	mob_trait = TRAIT_DUMB_CUM
+	gain_text = span_notice("You feel an insatiable craving for seminal fluids.")
+	lose_text = span_notice("Cum didn't even taste that good, anyways.")
 	medical_record_text = "Patient seems to have an unhealthy psychological obsession with seminal fluids."
-	var/craving_after = 15 MINUTES
+	mood_quirk = TRUE
 	var/timer
+	var/timer_trigger = 15 MINUTES
 
-/datum/quirk/dumb4cum/on_spawn()
-	. = ..()
-	timer = addtimer(CALLBACK(src, .proc/crave), craving_after, TIMER_STOPPABLE)
+/datum/quirk/dumb4cum/add()
+	// Set timer
+	timer = addtimer(CALLBACK(src, .proc/crave), timer_trigger, TIMER_STOPPABLE)
 
-/datum/quirk/dumb4cum/proc/crave()
-	var/list/hungry_phrases = list(
-									"Your stomach rumbles a bit and cum comes to your mind.",\
-									"Urgh, you should really get some cum...",\
-									"Some jizz wouldn't be so bad right now!",\
-									"You're starting to long for some more cum..."
-								  )
-	to_chat(quirk_holder, span_love("[pick(hungry_phrases)]"))
+/datum/quirk/dumb4cum/remove()
+	// Remove status trait
+	REMOVE_TRAIT(quirk_holder, TRAIT_DUMB_CUM_CRAVE, "dumb4cum")
+
+	// Remove penalty traits
+	ADD_TRAIT(quirk_holder, TRAIT_ILLITERATE, "dumb4cum")
+	ADD_TRAIT(quirk_holder, TRAIT_DUMB, "dumb4cum")
+	ADD_TRAIT(quirk_holder, TRAIT_PACIFISM, "dumb4cum")
```

Comment:
Wouldn't it be REMOVE_TRAIT ?

Reply:
Thank you for pointing out this oversight. It will be addressed in a future commit.